### PR TITLE
When life throws you a tomato, you have to cover it in dry ice.

### PR DESCRIPTION
**Pull request by nabobalis**
Trying to get sunpy working again.

New version of setuptools, I suspect has broken how setuptools_scm finds package data.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [build-system]
 requires = [
-         "setuptools",
-         "setuptools_scm",
+         "setuptools>=45,!=61.0.0",
```
Comment (nabobalis) [inline pyproject.toml:3]:
```suggestion
         "setuptools>=56,<61",
```

File: tox.ini [hunk 2]
```diff
@@ -5,7 +5,7 @@ envlist =
     codestyle
     base_deps
 requires =
-    setuptools >= 30.3.0
+    setuptools >= 45, !=61.0.0
```
Comment (nabobalis) [inline tox.ini:8]:
```suggestion
    setuptools >=56, <61
```

File: setup.cfg [hunk 3]
```diff
@@ -102,6 +102,12 @@ docs =
 [options.packages.find]
 exclude = sunpy._dev
 
+[options.package_data]
+sunpy = CITATION.rst
+sunpy.data = sunpyrc
+sunpy.data.test = *, */*
+sunpy.visualization.colormaps = */*
```
Comment (nabobalis) [inline setup.cfg:109]: I want to delete this.

**Review by ConorMacBride — APPROVED** [review]
This looks great.

**Comment by nabobalis** [comment]
> This looks great.

You know it.

**Pull request merged**